Marquee/Networking/JSONManager.swift

```diff
@@ -54,11 +53,10 @@ struct JSONManager {
     
     func parseJSON<T:Decodable>(_ movieData: Data) -> T? {
         let decoder = JSONDecoder()
+        decoder.keyDecodingStrategy = .convertFromSnakeCase
```

Comment:
Ahh I see you're converting from snake case, well done! This should work and you can ignore my previous comments about coding keys. FYI you might be able to write some good unit tests around this.